### PR TITLE
Added conda-build-all.

### PR DESCRIPTION
**Pull request by pelson**

### CODE REVIEW

File: recipes/conda-build-all/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package:
+  name: conda-build-all
+  version: "0.6.0"
+
+source:
+    git_url: git@github.com:SciTools/conda-build-all.git
```
Comment (ocefpaf) [inline recipes/conda-build-all/meta.yaml]: Would `https` be better here?

Comment (pelson) [inline recipes/conda-build-all/meta.yaml]: Yep. It looks like it failed with the ssh form. Doh!

**Comment by pelson** [comment]
Thanks @ocefpaf - I've merged this and will fix anything that crops up. I'm still playing with conda-build-all, but I expect to start looking at using it instead of obvious-ci for building at some point in the future.

**Comment by ocefpaf** [comment]
Looking forward to it!

**Pull request merged**